src/components/home/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+import { menuHeightOffset, Title1, SubTitle1, Button, Arrow } from '../styledComponents';
+import headerImage from '../../assets/homePage/headerImage.jpg';
+
+const HeaderContainer = styled(Grid)({
+  height: `${680 - menuHeightOffset}px`,
```

Comment:
Add a comment to explain this calculation - anytime there are "magic" numbers like this, it's not clear what this is trying to calculate just from reading this file.